### PR TITLE
Adapt GUI to render previous command

### PR DESCRIPTION
**Pull request by yl-ang**
1. Display the previous command in the command box when u + enter is provoked.
2. Display "Here is your previous command." in the command result box.

### CODE REVIEW

**Review by aditi2313 — APPROVED** [review]
LGTM! 💯

File: src/main/java/seedu/address/ui/CommandBox.java [hunk 1]
```diff
@@ -42,8 +42,15 @@ private void handleCommandEntered() {
...
 
         try {
-            commandExecutor.execute(commandText);
+            CommandResult commandResult = commandExecutor.execute(commandText);
+
+            if (commandResult.isPrevious()) {
+                final String previousCommand = commandResult.getFeedbackToUser().split("\\|+")[0];
+                commandTextField.setText(previousCommand);
+                return;
+            }
```
Comment (yuqitanyq) [inline src/main/java/seedu/address/ui/CommandBox.java:51]: I like how the regex with the greedy check is used here to ensure that even with multiple "|" used, the string can still be split correctly. 👍

**Review by yuqitanyq — APPROVED** [review]
LGTM 👍 Good job on integrating the GUI to the rendering of the previous command.

File: src/main/java/seedu/address/ui/CommandBox.java (hunk 1 above)
Comment (yl-ang) [reply src/main/java/seedu/address/ui/CommandBox.java:51]: Thank you :D

**Pull request merged**